src/_community/rules.md

```diff
@@ -46,14 +46,38 @@ Essentially, please take requests from community team members seriously and list
 something. If you have a problem with what a team member is asking you to do, we advise that you step away from the 
 situation and send a private message to another team member.
 
-#### 5) Keep all projects legal & appropriate
-
-If you need help with a project, you will not be provided with it if your project may break laws, breach a Terms of
-Service or EULA (belonging to any organisation, not just the platforms we're using), or be considered malicious or 
-inappropriate.
-
-Additionally, keep your code appropriate. Do not sexualize your code or use any slurs or other forms of derogatory
-language, either in the code itself or in the assets or filenames that make up your project.
+#### 5) Keep all projects legal, legitimate & appropriate
+
+While anyone may make use of the Quilt toolchain in accordance with the licenses applied to each project, all projects 
+developed or promoted in Quilt community spaces must meet the following criteria:
+
+* Projects must not be malicious - which includes (but is not limited to) malware, mods that include backdoors, or 
+  mods that harass specific people or groups of people (for example, by automatically kicking them from servers with 
+  the mod installed)
+* Projects must not be sexualised, hateful or otherwise inappropriate in nature, and this also applies to all code 
+  and project assets posted for any purpose
+* Projects must not break laws, including copyright laws - this includes piracy tools, as well as projects that 
+  ignore or otherwise break the licenses of other projects
```

Comment:
Should there be a disclaimer that some exceptions can be made given intention and method? (eg. Fukkit)

Reply:
Regarding the whole unavoidable GPL-doesn't-work-in-our-world thing?

I'm not confident that we can make licence exceptions on behalf of other people - that said, this part of the rule is intended for projects like Thallium (projects that ignore licensing-related requests) rather than projects that may or may not be stuck due to GPL-related technicalities that haven't yet been tested.

Reply:
Would it be possible for it to be mentioned? Like "Some gray-area instances may be acceptable on a case-by-case basis depending on context"

Reply:
Alternatively, add "intentional ignoring/breaking license" or "fails to adhere to licensing to the best of the project's ability"?

Reply:
That might be acceptable, honestly - I'll have a chat with the others

Reply:
Having had a chat, we feel like @Camotoy's approach here is more appropriate, otherwise the community team has to start deciding what breaks a licence and what doesn't - either way, I'll have a look

Reply:
It's less about deciding whether it does violate license and more about whether it's justified

Reply:
I don't think whether it's justified is really any of our business, especially considering that licences are legal documents - I've rephrased this block to the following in my local copy:

> * Projects must not break laws, including copyright laws - this includes piracy tools (including mods that use 
>  projects like `youtube-dl` for in-game music streaming), as well as projects that fail to adhere to the licenses of
>  other projects to the best of their ability

Reply:
Resolved, acknowledges imperfection & effort